rllib/examples/curriculum/atari_pong_curriculum_learning.py

```diff
@@ -0,0 +1,188 @@
+"""Example of using curriculum learning for Atari Pong by implementing a custom callback.
```

Comment:
Let's call this file `pong_curriculum_learning.py` to make it uniform with some of the `tuned_examples` AND with the corresponding callbacks file name (which also is just called `pong_...`).